exonum-java-binding-qa-service/src/test/java/com/exonum/binding/qaservice/QaServiceImplIntegrationTest.java

```diff
@@ -272,6 +276,29 @@ void getValueBeforeInit() {
     );
   }
 
+  @Test
+  void afterCommit() throws CloseFailuresException {
+    try (MemoryDb db = MemoryDb.newInstance();
+        Cleaner cleaner = new Cleaner()) {
+      Fork view = db.createFork(cleaner);
+      setServiceNode(node);
+      service.initialize(view);
+
+      BlockCommittedEvent event = mock(BlockCommittedEvent.class);
```

Comment:
Why do you mock a value and not just create it? You don't seem to verify any interactions with it

Reply:
But why can't we use this mock as a stub? We don't use it inside `afterCommit` method, so we can avoid creating it. What are the disadvantages of using a mock here (or advantages of creating an object)? Mock seems simpler to me.

Reply:
As a mocking framework is designed to verify interactions between objects, using a mock usually suggests that you will do that — verify the SuT interaction with this mock (or help it get through to the interesting condition). As one does not verify interactions with values, using mocks to create an object gives a misleading message.